source/slang/slang-type-layout.cpp

```diff
@@ -295,6 +295,92 @@ struct Std430LayoutRulesImpl : GLSLBaseLayoutRulesImpl
     // base/common GLSL layout rules.
 };
 
+/// GLSL fvk-use-dx-layout for `ShaderResource`
+struct FXCShaderResourceLayoutRulesImpl : DefaultLayoutRulesImpl
+{
+    // Currently this FXC layout is equal to how we compute 'DefaultLayoutRulesImpl'
+};
+
+/// GLSL fvk-use-dx-layout for `ConstantBuffer`
+struct FXCConstantBufferLayoutRulesImpl : DefaultLayoutRulesImpl
```

Comment:
Why can't this be just HLSLConstantBufferLayoutRules?

Reply:
We can use `HLSLConstantBufferLayoutRules`.